networkit/cpp/graph/Graph.h

```diff
@@ -645,6 +645,8 @@ class Graph final {
 	 */
 	void removeEdge(node u, node v);
 
+	void removeEdgesFromIsolatedSet(const std::vector<node> &nodesInSet);
```

Comment:
This method should have a comment to describe what it does. You can just copy the one from the Python interface.

Reply:
Addressed in f096e901f296a2e293463c6e32f910172fcd3882